src/test/java/uk/gov/hmcts/reform/pdf/service/domain/GeneratePdfRequestTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.pdf.service.domain;
+
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Base64;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Confirm Jackson serialisation assumptions.
+ */
+public class GeneratePdfRequestTest {
+
+    private ObjectMapper mapper = new ObjectMapper();
+
+    @Test(expected = JsonMappingException.class)
+    public void template_key_is_required() throws IOException {
+        String json = "{\"values\":{}}";
```

Comment:
`assertByThrownBy(() -> ...)` would allow us to inspect the exception and check if the cause is what we expect it to be. Just to consider.

Reply:
Thanks @lgonczar, I'll consider it in the future. Here it would look bit obscure as test method's name clearly states its purpose, contains one assertion and one line of tested code.

Reply:
The reason why I mentioned this assertion is that we can't be sure if JsonMappingException was thrown because of missing template key or because of some other problem, e.g. malformed JSON.